app/src/main/java/ai/elimu/appstore/util/VersionHelper.java

```diff
@@ -54,7 +54,15 @@ public static void updateAppVersion (Context context) {
                 }
             }
 
-//            if (oldVersionCode < 2000012) {
+            if (oldVersionCode < 2000018) {
```

Comment:
Why don't you merge 2 if conditions into 1 only: if (oldVersionCode < 2000018) ?

Reply:
Fixed